assets/onboarding/features/index.jsx

```diff
@@ -113,6 +114,7 @@ const Features = () => {
 	};
 
 	const goToNextStep = () => {
+		completeSetupWizard();
```

Comment:
I'm wondering if it would make more sense doing it while loading the ready step, using the same `submitStep` from the `useSetupWizardStep`. WDYT?

We could also make a check if the ready step is already completed to don't submit if the user navigate back in the steps and go to the ready again. This check, maybe we could do while working in this issue: #3135

Reply:
> I'm wondering if it would make more sense doing it while loading the ready step,

This makes sense to me, as it would more easily account for the possibility of injecting a new step between Features and Ready.

Reply:
I'll remove this from here and add the call in #3135, as it can be better handled with some state management around steps. Running it every time the Ready component is mounted is a bit messy. - 9bab849b

Reply:
Should we remove also the related actions?

Reply:
The functionality and the REST API endpoint is still part of this PR, so I think it's frontend action can stay too.